### ISSUE TITLE
MedicationRequest - Proactively fetch included Medications 

### ISSUE THREAD

**Issue by kmarx** [drajer-health/eCRNow#406]
`Medication` resources take up more than 65% of the eCRNow FHIR request volume in our EHR. Can eCRNow request them all at once with the initial `MedicationRequest` instead of as References ? E.g.,
```
MedicationRequest?patient={{patientPublicid}}&_include=MedicationRequest:medication
```
If not universally desirable, perhaps this behavior could be toggled via a new application.property

**Comment by nbashyam** [comment]
The commit message to look for is Fixes for #422 and #421

**Comment by nbashyam** [comment]
This is implemented in the baseline already.
